fe/fe-common/src/main/java/org/apache/doris/common/Config.java

```diff
@@ -2165,4 +2171,17 @@ public class Config extends ConfigBase {
                     + "The larger the value, the more uniform the distribution of the hash algorithm, "
                     + "but it will increase the memory overhead."})
     public static int virtual_node_number = 2048;
+
+    @ConfField(description = {"控制对大表的自动ANALYZE的最小时间间隔",
+            "Control the minimum time interval for automatic ANALYZE on large tables"})
+    public static long huge_table_auto_analyze_interval_in_millis = TimeUnit.HOURS.toMillis(12);
+
+    @ConfField(description = {"定义大表的大小下届", "Define the size of the large table"})
```

Comment:
```suggestion
    @ConfField(description = {"定义大表的大小下界", "Define the size of the large table"})
```
btw, this desc is not clear, when i saw it, i do not know it use for stats collection and what't the huge table means

Reply:
I think we should explain what huge table means is the documentation rather than here